src/mmw/apps/geoprocessing_api/views.py

```diff
@@ -1344,6 +1344,319 @@ def start_analyze_drb_2100_land(request, key=None, format=None):
     ], area_of_interest, user, messages=msg)
 
 
+@swagger_auto_schema(method='post',
+                     request_body=schemas.ANALYZE_REQUEST,
+                     responses={200: schemas.JOB_STARTED_RESPONSE})
+@decorators.api_view(['POST'])
+@decorators.authentication_classes((SessionAuthentication,
+                                    TokenAuthentication, ))
+@decorators.permission_classes((IsAuthenticated, ))
+@decorators.throttle_classes([BurstRateThrottle, SustainedRateThrottle])
+@log_request
+def start_analyze_drainage_area(request, format=None):
```

Comment:
This doesn't seem to be available in swagger right now. Are we going to add it later?
<img width="382" alt="image" src="https://user-images.githubusercontent.com/60887686/180931441-d89c2e23-b6b7-4092-b0f9-796e19db1891.png">

Reply:
It is available in the `/analyze/` section:

![image](https://user-images.githubusercontent.com/1430060/181036534-1826f7eb-8ef2-4937-9ff2-d90f12d8410f.png)